.github/workflows/google_chat_caller.yml

```diff
@@ -1,10 +1,10 @@
 name: PullRequestNotifier
 
-on:
-  pull_request:
-    types: [opened, edited, closed, synchronize, review_requested, ready_for_review, labeled]
-    branches:
-      - "**"
+#on:
```

Comment:
If you remove all triggers - as far as I understood - the action will never be triggered. So why not just remove the whole action?

Otherwise I would say instead of commenting it, just remove the whole block

Reply:
ok, i removed the whole file.